include/dlpack/dlpack.h

```diff
@@ -134,10 +136,11 @@ typedef enum {
  * export an array with non-native endianness
  *
  *  Examples
- *   - float: type_code = 2, bits = 32, lanes=1
- *   - float4(vectorized 4 float): type_code = 2, bits = 32, lanes=4
- *   - int8: type_code = 0, bits = 8, lanes=1
+ *   - float: type_code = 2, bits = 32, lanes = 1
+ *   - float4(vectorized 4 float): type_code = 2, bits = 32, lanes = 4
+ *   - int8: type_code = 0, bits = 8, lanes = 1
  *   - std::complex<float>: type_code = 5, bits = 64, lanes = 1
+ *   - bool: type_code = 6, bits = 8, lanes = 1
```

Comment:
would be good to add more comments: bool comes with bits=8 as it is normally the underlying storage type of common array libraries.

Reply:
Done in [fd9b858](https://github.com/dmlc/dlpack/pull/114/commits/fd9b85822816caee1da480a300d841f0ff6f6e5b).

Reply:
Updated again.